src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -11,10 +11,12 @@ const MAX_REPOS_RETURNED = 20;
 
 export const GithubCreateBranchGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const {
-		jiraHost,
+		jiraHost: jiraHostLocal,
```

Comment:
open to opinions on naming...

But the jirahost could come from two places, 1) - the session.. yay 2) query param....